.env.faf-stack

```diff
@@ -0,0 +1,33 @@
+# node env is set statically to "development" in the dockerfile, you could override it here.
```

Comment:
What was the thinking with having both a .env.template and .env.fafstack? It seems a little redundant

Reply:
env.example is for local development (without docker and a faf-stack)